packages/cattle-pushprox/Chart.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+name: cattle-pushprox
+description: A Rancher chart based on PushProx that sets up a deployment of the PushProx proxy and several PushProx clients on a Kubernetes cluster.
```

Comment:
Can this be clarified? What does 'several' mean?

Reply:
Sure, will change it to `a DaemonSet of PushProx clients`